src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs

```diff
@@ -264,6 +263,7 @@ internal string FindBestVersionMatch(VersionRange versionRange, IEnumerable<stri
                 return version;
             }).Where(v => v != null).OrderByDescending(version => version.Version).ToList();
 
+            _logger.LogInformation("Platform release channel is set to: {platformReleaseChannelVersion}. Attempting to resolve bundle version for {bundleId} using the platform release channel configuration.", _platformReleaseChannel, bundleId);
```

Comment:
Note: This value will be set by a specific subset of customers, but most won't use it. All logs related to platform release channel are `Information` level to avoid concern.

Reply:
> but most won't use it

Can we only log when it is used to avoid noise?